base/strings/string.jl

```diff
@@ -341,6 +341,10 @@ julia> repeat('A', 3)
 ```
 """
 repeat(c::AbstractChar, r::Integer) = repeat(Char(c), r) # fallback
+#don't assume effects for general integers since they may be all sorts of screwed up.
```

Comment:
```suggestion
# don't assume effects for general integers since we cannot know their implementation
```

or something like that. "all sorts of screwed up" sounds inelegant.